src/actions/content.js

```diff
@@ -150,6 +150,12 @@ function* loadSingleContent(action) {
   }
 }
 
+export const CONTENT_ADD_CHANGE = 'CONTENT_ADD_CHANGE';
```

Comment:
Just from quickly taking a look, I'm not sure this name feels correct. I'm not totally sure what is happening just from this action. Not sure if that makes sense?

Reply:
Yeah, so action is for storing the state of the actual node add form

Reply:
Do you have a better suggestion? This is basically about storing node form content in redux